pulseapi/entries/views.py

```diff
@@ -22,7 +22,7 @@
 
 
 @api_view(['PUT'])
-def toggle_bookmark(request, entryid):
+def toggle_bookmark(request, entryid, **kwargs):
```

Comment:
can we make this a named argument instead, or does this *have* to be kwargs?

Reply:
`kwargs` is cleaner otherwise I'll have to add flake8 exceptions for each line since we aren't using the argument anywhere in the function